### ISSUE TITLE
TheHive Docker Install - Cannot connect to elasticsearch.

### ISSUE THREAD

**Issue by mmcabee** [TheHive-Project/TheHiveDocs#71]
Howdy,

Trying to test TheHive and I am installing via docker. I have installed both Docker-CE and Docker-Compose, and The Hive successfully runs and I can get to the login page. However, it does not prompt me to make an admin account, and seems to be failing to connect to elasticsearch as I am incapable of cating any indicies in a seperate PuTTY window. 

Any help will be greatly appreciated. Thanks!

**Comment by tonygumbrell** [comment]
Make sure your Hive configuration file (/etc/configuration.yml) has the DNS name of your elastic container setup under the elastic stanzas.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by mmcabee** [comment]
So I actually figured out what the issue was.

I was trying to construct The Hive in a dev environment with drastically less system resources than needed. Once enough resources were given to the system, I noticed in the "up" output that Elasticsearch kept dying because the kernel's default max virtual machine memory was too low (it mentioned a value of requiring 262144). This is fixed by modifying the "/etc/sysctl.conf" file and creating a new uncommented line that contains "vm.max_map_count=262144". After reboot, all components were able to start in their docker container and The Hive worked like a charm.

Thanks!
